src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -106,6 +109,12 @@ class BookStep : public StepImp<TIn, TOut, BookStep<TIn, TOut>>
         return EitherAmount (cache_->out);
     }
 
+    bool
+    redeems (ReadView const& sb, bool fwd) const override
+    {
+        return !ownerPaysTransferFee_;
```

Comment:
I'm surprised that this is the calculation, not saying that you're wrong.  Intuitively, it seems like `redeems` should take a look in the ledger?  Isn't `ownerPaysTransferFee_` just the state of the feature, or did I miss an extra modification while the flag was being passed down?